pf9-express

```diff
@@ -580,15 +580,15 @@ fi
 ## build group_vars/all.yml
 build_config --skip-inventory-check
 
-# validate all config values are set
```

Comment:
This should remain in it's original spot. The modification you are looking for starts on new line 642 through 654 (possibly).